layers/+lang/python/packages.el

```diff
@@ -17,7 +17,6 @@
     company-anaconda
     cython-mode
     eldoc
-    ;; evil-matchit
```

Comment:
It is included in the [spacemacs distibution](https://github.com/syl20bnr/spacemacs/blob/develop/layers/%2Bdistribution/spacemacs/packages.el#L38).